packages/terra-functional-testing/src/services/wdio-terra-service/GithubPr.js

```diff
@@ -0,0 +1,10 @@
+const GithubIssue = require('./GithubIssue');
+
+class GithubPr extends GithubIssue {
+  async getBaseBranchRef() {
```

Comment:
Only to follow the one-class-per-file rule that the linter defaults too.

Reply:
Also Issues don't have this branch data. PRs do though. According to the GitHub API doc, every PR is an Issue, but not every Issue is a PR. So that's why I have them as two separate classes.

Reply:
Thanks for the clarifications!